gravitee-am-ui/src/app/domain/settings/certificates/certificates.component.ts

```diff
@@ -87,14 +88,12 @@ export class DomainSettingsCertificatesComponent implements OnInit {
     return (cert.status == 'expired')
   }
 
-  expireInDays(expiry) {
-    return Math.ceil( (expiry - Date.now()) / (1000 * 3600 * 24));
+  certificateIsRenewed(cert) {
+    return (cert.status == 'renewed')
```

Comment:
Typescript uses the same equality rules as Javascript, so it's generally best practice to use `===` over `==`. 

```suggestion
    return cert.status === 'renewed';
```